packages/web-components/src/components/button/button.stories.ts

```diff
@@ -301,6 +308,18 @@ export const IconButton = {
     </cds-button>`,
 };
 
+export const iconButtonWithBadge = {
+  argTypes: controls,
+  args: defaultArgs,
+  render: ({ badgeCount }) =>
```

Comment:
think we need to add the `disabled` arg here since I'm still seeing the badge count when I have the button disabled in storybook

Reply:
thanks updated!